src/helpers/text/purify.js

```diff
@@ -0,0 +1,17 @@
+import DOMPurify from "dompurify";
+
+function purify(dirtyText, useProfiles = { html: false }) {
+    if (typeof dirtyText !== "string") {
+        throw new Error("Input must be a string");
+    }
+
+    try {
+        const purifiedText = DOMPurify.sanitize(dirtyText, { USE_PROFILES: useProfiles });
```

Comment:
what's the use of useProfiles here?

Reply:
Here, the `useProfiles` parameter offers precise control over DOMPurify's HTML handling. When `html` is `true`, it allows safe HTML elements and attributes while removing malicious content, resulting in purified HTML. On the other hand, when `html` is set to `false`, it treats the input as plain text, removing HTML tags and attributes to produce plain text output.